### ISSUE TITLE
window is not defined

### ISSUE THREAD

**Issue by n-sviridenko** [redux-offline/redux-offline#34]
Hello! I have the following error when I'm using it with https://github.com/kriasoft/react-starter-kit . Maybe you need to check if `window` is not `undefined` or provide it via `global` variable?

```
ReferenceError: window is not defined

  - detectNetwork.js:21 Object.exports.default [as detectNetwork]
    [livery-client-react]/[redux-offline]/lib/defaults/detectNetwork.js:21:7

  - index.js:56 createOfflineStore
    [livery-client-react]/[redux-offline]/lib/index.js:56:12
```

**Comment by n-sviridenko** [comment]
It could be because of the server side rendering

**Comment by n-sviridenko** [comment]
Any news?

**Comment by jevakallio** [comment]
@n-sviridenko apologies for radio silence, took a while to clear time to get back to working on this library.

This is fixed by #47 - I will release a new version shortly
